CLAUDE.md

```
@@ -0,0 +1,132 @@
+신규 env 변수 추가 시 `.env.example`에도 키 반영.
+
+## GitHub Conventions
+
+### Branch
+
+`{이슈번호}-{type}-{설명-kebab-case}` — 예: `154-feat-에러페이지-구현`
+
+### Commit Message
+
+`type: 한국어 설명
+타입: `feat`, `fix`, `refactor`, `chore`, `docs`, `style`, `test`
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)